testSuite/scripts/test_upload_page_blob.py

```diff
@@ -39,44 +39,76 @@ def test_page_blob_upload_1mb_with_oauth(self):
     # test_page_range_for_complete_sparse_file verifies the number of Page ranges for
     # complete empty file i.e each character is Null character.
     def test_page_range_for_complete_sparse_file(self):
+        # step 1: uploading a sparse file should be optimized
```

Comment:
Do we have a similar test for downloading?  Can we add a backlog item to create such a test next week?

Reply:
I'll add it now